README.md

```diff
@@ -1,5 +1,3 @@
-# learning-git
-
 **Preconditions:** 
 -  Create Word or any other document where you are going to collect all commands that you are going to type in these exercises below. That document will help us to track your work and solutions, so don't forget to save it. 
```

Comment:
You shouldn't have any changes to this file, so you should revert your changes here.